### PR TITLE
CCAAS Builders

### PR DESCRIPTION
**Pull request by mbwhite**
- adjust the makefile to have targets for linux-amd64/darwin-amd64/windows-amd64
- adjust the packaging as suggested in issue 3286

Signed-off-by: Matthew B White <whitemat@uk.ibm.com>

#### Type of change

- Bug fix
- New feature
- Improvement (improvement to code, performance, etc)
- Test update
- Documentation update

### CODE REVIEW

**Comment by mbwhite** [comment]
resolves issue #3286

**Review by jt-nti — DISMISSED** [review]

**Comment by mbwhite** [comment]
@denyeart this is good for review :-)

File: integration/externalbuilders/binary/bin/detect [hunk 1]
```diff
@@ -18,5 +18,7 @@ if [ "$(jq -r .type "$META/metadata.json")" == "binary" ]; then
     exit 0
 fi
 
+>&2 cat $META/metadata.json
```
Comment (jt-nti) [inline integration/externalbuilders/binary/bin/detect:21]: Did you mean to leave this in?

Comment (mbwhite) [reply integration/externalbuilders/binary/bin/detect:21]: well I didn't but then it was useful in debugging - as later on the peer lies about what the language really is.
(i.e. it toUpperCase() the language - so I first assumed the packaging of the contract was wrong

**Review by jt-nti — APPROVED** [review]

**Review by denyeart — APPROVED** [review]

**Comment by denyeart** [comment]
@Mergifyio backport release-2.5

**Comment by denyeart** [comment]
@Mergifyio backport release-2.4

**Pull request merged**